.github/workflows/notebook-bot.yml

```diff
@@ -4,24 +4,28 @@ on:
   pull_request_target:
```

Comment:
Alex mentioned we should replace this with `pull_request` whenever possible. Got any insight on that perchance?

Reply:
Will check with @alexnorell

Reply:
Yeah, this needs to be refactored to `pull_request`